packages/block-editor/src/components/iframe/use-compatibility-styles.js

```diff
@@ -45,6 +45,11 @@ export function useCompatibilityStyles() {
 					return accumulator;
 				}
 
+				// Don't try to add global style presets; they're directly rendered in iframe.
+				if ( ownerNode.dataset.globalStyle === 'preset' ) {
```

Comment:
Alternatively we could only accept stylesheets with an ID, which is any stylesheet added through the WP dependency system. What do you think?

Reply:
That should work as well, and no need to maintain the "markers" concept. I'll update the code.

Reply:
Great. This compat layer was anyway only intended to be for styles added through the WP dependency system.